src/pages/index.js

```diff
@@ -97,3 +208,28 @@ profileEditButton.addEventListener("click", () => {
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
+
+profileImageContainer.addEventListener("click", () => {
+  avatarPopup.open();
+  avatarFormValidator.resetValidation();
```

Comment:
`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs. 


You should use `disableButton` in a `then` block after submission (not on opening)